### PR TITLE
Add bridging unregistered traffic IPv6 test

### PR DESCRIPTION
**Pull request by stavrukPLV**
**This includes:**
- test_bridging_unregistered_traffic_ipv6

### CODE REVIEW

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    address_map = (
+        # swp port, tg port,    tg ip,       gw,          plen
+        (ports[0], tg_ports[0], "2001:2::2", "2001:1::1", 64),
+        (ports[1], tg_ports[1], "2001:3::2", "2001:1::1", 64),
+        (ports[2], tg_ports[2], "2001:4::2", "2001:1::1", 64),
+        (ports[3], tg_ports[3], "2001:5::2", "2001:1::1", 64),
+    )
+
+    dev_groups = tgen_utils_dev_groups_from_config(
+        {"ixp": port, "ip": ip, "gw": gw, "plen": plen}
```
Comment (SerhiyBoikoPLV) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py]:
```suggestion
        {"ixp": port, "ip": ip, "gw": gw, "plen": plen, "version": "ipv6"}
```

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcMac": {"type": "increment",
+                       "start": "00:00:00:00:00:35",
+                       "step": "00:00:00:00:10:00",
+                       "count": mac_count},
+            "dstMac": "aa:bb:cc:dd:ee:11",
+            "frameSize": 78,
+            "type": "raw",
+            "protocol": "ipv6",
```
Comment (SerhiyBoikoPLV) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py:101]: srcIp? dstIp?

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    10. Enable back multicast flooding on the ports.
+    11. Send increment traffic from TG.
+    12. Verify that traffic flooded to all ports that are members in br0.
+    """
+
+    bridge = "br0"
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        print("The testbed does not have enough dent with tgen connections")
+        return
```
Comment (SerhiyBoikoPLV) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py]:
```suggestion
    if not tgen_dev or not dent_devices:
        pytest.skip("The testbed does not have enough dent with tgen connections")
```

**Review by SerhiyBoikoPLV — CHANGES_REQUESTED** [review]

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    11. Send increment traffic from TG.
+    12. Verify that traffic flooded to all ports that are members in br0.
+    """
+
+    bridge = "br0"
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        pytest.skip("The testbed does not have enough dent with tgen connections")
+    dent_dev = dent_devices[0]
+    device_host_name = dent_dev.host_name
```
Comment (yuriilisovskyi) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_unregistered_traffic_ipv6.py]: dent_dev is not used later, change it to `device_host_name = dent_devices[0].host_name`

**Review by yuriilisovskyi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**